Makefile

```diff
@@ -18,8 +18,11 @@ docker-nearcore-nightly:
 	docker build -t nearcore-nightly -f Dockerfile --build-arg=features=nightly_protocol,nightly_protocol_features --progress=plain .
 
 release: NEAR_RELEASE_BUILD=release
```

Comment:
There’s also this environment variable which we’re using to check if release binary is used on mainnet. We probably should convert it into use of a feature as well. So maybe `release_build` feature in `neard`?